connectomics/data/utils/data_io.py

```diff
@@ -23,10 +23,16 @@ def readvol(filename, dataset=''):
         raise ValueError('unrecognizable file format for %s'%(filename))
     return data
 
-def savevol(filename, vol, dataset='main'):
-    fid = h5py.File(filename, 'w')
-    fid.create_dataset(dataset, data=vol, compression='gzip')
-    fid.close()
+def savevol(filename, vol, dataset='main', format='h5'):
```

Comment:
I have a doubt here. The filename specifies where you need to save the vol. For ex: `/n/home/test.h5`
If I'm using the PNG format, I don't use this information. Plus, the filename needs to be passed as a parameter in the `process_syn.py` I guess. Let me know if there's anything I can do about this.